data/templates/affiliations.jats

```diff
@@ -0,0 +1,35 @@
+$--
+$-- Affiliations
+$--
+$for(affiliation)$
+<aff id="aff-$affiliation.id$">
+$-- wrap affiliation if it has a known institution identifier
```

Comment:
Should the `<institution-wrap>` opening line be below this comment, before all the `institution` elements?

Also it looks like `<institution-wrap>` is always added, without testing for the presence of an identifier (which is fine, just disagrees with the comment).

Reply:
I thought about doing that. The reason is that `<institution-wrap>`, IIUC, cannot contain PCDATA, so we cannot use commas to separate the different `<institute>` elements as done in the current version.

The comment is indeed outdated, thanks for pointing that out.